deploy/charts/approver-policy/values.yaml

```diff
@@ -152,6 +152,18 @@ hostNetwork: false
 # This value may need to be changed if `hostNetwork: true`
 dnsPolicy: ClusterFirst
 
+# Configure the priority class of the Pod.
+#
+# For more information, see:
+# * [Guaranteed Scheduling For Critical Add-On Pods](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/)
+# * [Protect Your Mission-Critical Pods From Eviction With PriorityClass](https://kubernetes.io/blog/2023/01/12/protect-mission-critical-pods-priorityclass/)
+#
+# For example:
+#   priorityClassName: system-cluster-critical
+#
+# +docs:property
+# priorityClassName: system-cluster-critical
```

Comment:
```suggestion
# priorityClassName: ""
```

I am under the impression that the empty string is "falsy" in Helm, and you've added a conditional around setting the field in the Deployment. And I prefer to see the value defaults.

Reply:
Setting the default value to `null` might also work well.